### PR TITLE
move more generic utils in here (i.e. those used in plone.app.content)

### PR DESCRIPTION
**Pull request by jensens**
Lots of stuff which slipped through, but nothing heavy.

### CODE REVIEW

File: src/plone/base/utils.py [hunk 1]
```diff
@@ -36,25 +61,25 @@ def human_readable_size(size):
     return f"{float(size / float(SIZE_CONST[c])):.1f} {c}"
 
 
-def safeToInt(value, default=0):
+def safe_to_int(value, default=0):
```
Comment (thet) [inline src/plone/base/utils.py:64]: +1 
I was just to propose a name change but now it's done :D
Only, maybe we should rename it to ``safe_int`` to be closer to the other names we are using here: ``safe_text``, ``safe_callable``, etc.

**Review by thet — APPROVED** [review]
Now as the utils.py module is growing, should we move it to a separate subpackage and split it up?
That could also be done in another PR.
Also I have a comment about renaming ``safe_to_int`` to ``safe_int`` but that's not a dealbreaker.
So, approved.

File: src/plone/base/utils.py (hunk 1 above)
Comment (jensens) [reply src/plone/base/utils.py:64]: I tried to not change it as much, but want to get  rid of camel case. In fact I first had it w/o `to`. Well, I am +/- 0 for both.

Comment (jensens) [reply src/plone/base/utils.py]: ok, changed and added deprecation, because I used the form already. Just to be sure.

**Pull request merged**